fastlane/swift/Fastlane.swift

```diff
@@ -2962,7 +2962,7 @@ func setChangelog(appIdentifier: String,
                                          description: String? = nil,
                                          isDraft: Bool = false,
                                          isPrerelease: Bool = false,
-                                         uploadAssets: String? = nil) -> [String : String] {
+                                         uploadAssets: [String]? = nil) -> [String : String] {
```

Comment:
This change will automatically happen at the next release when the release script regenerates the Swift implementation.

Reply:
Ahh perfect